# gps relative position
string timestamp
float64 delx
float64 dely
float64 longitude
float64 latitude
